src/Asset.js

```diff
@@ -200,7 +200,7 @@ class Asset {
       const packageName = sanitizeFilename(this.package.name, {
         replacement: '-'
       });
-      return packageName + ext;
+      return md5(packageName) + ext;
```

Comment:
@devongovett is all I need to do in this function? My intent is to not do any renaming on the entry file: https://github.com/parcel-bundler/parcel/pull/829/files#diff-784c5f1cd4c785db2834e618a52b438eR35